libraries/AP_DroneCAN/AP_DroneCAN.cpp

```diff
@@ -1056,16 +1056,21 @@ void AP_DroneCAN::gnss_send_yaw()
 // SafetyState send
 void AP_DroneCAN::safety_state_send()
 {
-    uint32_t now = AP_HAL::millis();
-    if (now - _last_safety_state_ms < 500) {
-        // update at 2Hz
+    const uint32_t now = AP_HAL::millis();
```

Comment:
we should send at 10Hz for the first 1000ms after a change to arm or safety state

Reply:
can use hal.util->get_last_armed_change()

Reply:
I set it to 20Hz on-change for 1000ms per the chatter on the devcall. Personally, I think 10Hz is plenty.

Reply:
however, @peterbarker pointed out that the immediate on-change send functionality is lost

Reply:
I can either store the _ms and use up 4 bytes of RAM, or check if the last_ms is <=2. I'm going with the <=2